tfjs-backend-webgpu/src/kernels/from_pixels_webgpu.ts

```diff
@@ -26,35 +26,123 @@ export class FromPixelsProgram implements WebGPUProgram {
   variableNames = ['A'];
   dispatchLayout: {x: number[]};
   dispatch: [number, number, number];
+  pipeline: GPUComputePipeline;
+  bindGroupLayout: GPUBindGroupLayout;
+
+  uniform: GPUBuffer;
+  lastUniformContent = [0, 0, 0];
+
+  inputTexture: GPUTexture = null;
+  lastPixelSize = {width: 0, height: 0};
 
   constructor(outputShape: number[]) {
     this.outputShape = outputShape;
     this.dispatchLayout = flatDispatchLayout(this.outputShape);
     this.dispatch = computeDispatch(this.dispatchLayout, this.outputShape);
-
+    const workGroupSize = 256;
```

Comment:
just curious - how did you arrive at this number?

Reply:
Haha, it is from Jiajia's suggestion :p

Reply:
Sounds good :) Could you include a brief comment that you empirically determined this to be the best value?